### PR TITLE
#165441913 Get all reports by a user

### PR DESCRIPTION
**Pull request by NedyUdombat**
#### What does this PR do?
Get all reports by a user

#### Description of Task to be completed?
- [x] Get all articles reported by a user

#### How should this be manually tested?
- Clone the repo,
- Run ```npm install``` to install all dependencies,
- Create a .env file using the .env.example as a guide and enter your environment variables
- Ensure you have a PostgreSQL server running on your development environment
- Run `sequelize db:create`  
- Run `sequelize db:migrate` 
- Run `sequelize db:seed:all`   
- Run `npm run start:dev`
- Hit http://localhost:3000/api/v1/users/reports via your REST API client (postman or insomnia).
- You should see the response below

#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?(if applicable)
#165441913

#### Screenshots
Screenshot 2019-04-23 at 4 10 08 PM

### CODE REVIEW

File: src/routes/users.route.js [hunk 1]
```diff
@@ -97,4 +98,33 @@ userRouter.get(
   getReadingStats,
 );
 
+/**
+ * @swagger
+ *
+ * /api/v1/users/profiles/:username/stats:
```
Comment (igbominadeveloper) [inline src/routes/users.route.js]: your documentation is showing for `stats`

Comment (NedyUdombat) [reply src/routes/users.route.js]: Thank you, this has been resolved

File: src/tests/routes/controllers/users.test.js [hunk 2]
```diff
@@ -190,4 +190,17 @@ describe('User stats', () => {
...
+
+  context('User Reports', () => {
+    it('should return a 200 and get articles a user has reported', async () => {
+      const { status, body } = await chai
+        .request(app)
+        .get('/api/v1/users/reports')
+        .set('x-access-token', userToken);
+
+      expect(status).to.eql(200);
+      expect(body).to.have.keys('status', 'message', 'data');
```
Comment (Japhethca) [inline src/tests/routes/controllers/users.test.js:202]: let's test for the response data.

Comment (NedyUdombat) [reply src/tests/routes/controllers/users.test.js:202]: This has been resolved

**Review by Japhethca — APPROVED** [review]

**Pull request merged**